bower.json

```diff
@@ -30,7 +30,7 @@
     "ng-device-detector": "1.1.9",
     "ng-file-upload": "7.0.17",
     "ng-file-upload-shim": "~7.0.x",
-    "oi.multiselect": "0.2.2",
+    "oi.select": "0.2.21",
```

Comment:
https://github.com/tandibar/oi.multiselect (most recent release: Jan 2015) was forked from https://github.com/tamtakoe/oi.select (most recent release: May 2016). The latter is a far more active and feature rich.